Decred Wallet/Assets.xcassets/nav_menu/ic_receive.imageset/Contents.json

```diff
@@ -0,0 +1,23 @@
+{
+  "images" : [
+    {
+      "idiom" : "universal",
+      "filename" : "ic_receive_24px.png",
```

Comment:
Remove the `_24px`. Apply to all other icons.

Reply:
already renamed in later commits.

Reply:
resolved in 37e2453